interface/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "solana-stake-interface"
+version = "0.1.0"
+description = "Instructions and constructors for the Stake program"
+repository = "https://github.com/solana-program/stake"
+edition = "2021"
+readme = "README.md"
+license-file = "../LICENSE"
+
+[dependencies]
+borsh = { version = "1.5.1", features = ["derive", "unstable__schema"], optional=true }
+borsh0-10 = { package = "borsh", version = "0.10.3", optional=true }
+num-traits = "0.2"
+serde = { version = "1.0.210", features = ["derive"] }
+serde_derive = "1.0.210"
+solana-decode-error = "^2.1"
+solana-clock = "^2.1"
+solana-cpi = "^2.1"
+solana-frozen-abi = { version = "^2.1", features = ["frozen-abi"], optional = true }
+solana-frozen-abi-macro = { version = "^2.1", features = ["frozen-abi"], optional = true }
+solana-instruction = { version = "^2.1", features = ["bincode", "std"] }
+solana-program-error = "^2.1"
+solana-pubkey = { version = "^2.1", default-features = false, features = ["serde"] }
+solana-system-interface = { version = "0.0.1", git = "https://github.com/solana-program/system.git" }
```

Comment:
side note: is this ready to be published so that we can use the version from crates.io instead?